packages/backend/src/tests/ledgerTransfer.ts

```diff
@@ -0,0 +1,45 @@
+import { v4 as uuid } from 'uuid'
+import { TransactionOrKnex } from 'objection'
+import {
+  LedgerTransfer,
+  LedgerTransferState,
+  LedgerTransferType
+} from '../accounting/psql/ledger-transfer/model'
+
+interface CreateLedgerTransferArgs {
+  transferRef?: string
+  amount?: bigint
+  ledger: number
+  creditAccountId: string
+  debitAccountId: string
+  state?: LedgerTransferState
+  type?: LedgerTransferType
+  expiresAt?: Date
+}
+
+export const createLedgerTransfer = async (
```

Comment:
For now just using the `LedgerTransfer` model directly, will switch over to using `accountingService` once transfers are built out